src/Command/CopyLayoutsCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+declare(strict_types=1);
+
+namespace BootstrapUI\Command;
+
+use Cake\Command\Command;
+use Cake\Console\Arguments;
+use Cake\Console\ConsoleIo;
+use Cake\Console\ConsoleOptionParser;
+use Cake\Core\Plugin;
+use Cake\Filesystem\Folder;
+
+/**
+ * Copies the sample layouts into the application's layout templates folder.
+ */
+class CopyLayoutsCommand extends Command
+{
+    /**
+     * @inheritDoc
+     */
+    public function execute(Arguments $args, ConsoleIo $io)
+    {
+        $io->info('Copying sample layouts...');
+
+        $layoutDir = new Folder(Plugin::path('BootstrapUI') . 'templates' . DS . 'layout' . DS . 'examples');
```

Comment:
Would be nice to avoid the use of deprecated `Folder` and `File` classes but okay if they are.

Reply:
Any idea why `\Cake\Filesystem\Filesystem` is internal?

I could certainly rewrite everything to use vanilla PHP, but in the end it would pretty much just recreate the functionality that CakePHP ships with, which feels kinda redundant.

Reply:
Just so that we won't have to provide BC promise for it's public API :)  It's unlikely to change though and we can use it in BUI.

Reply:
Alright, I'll switch things to use `Filesystem` and vanilla PHP then.

Reply:
done